lib/tools/adb-commands.js

```diff
@@ -1541,12 +1541,14 @@ methods.screenrecord = function (destination, options = {}) {
     cmd.push('--bugreport');
   }
   cmd.push(destination);
-  log.debug(`Building screenrecord process with the command line: adb shell ${cmd.join(' ')}`);
-  return new SubProcess(this.executable.path, [
+
+  const fullCmd = [
     ...this.executable.defaultArgs,
     'shell',
     ...cmd
-  ]);
+  ];
+  log.debug(`Building screenrecord process with the command line: ${fullCmd.join(' ')}`);
```

Comment:
why not use a shlex library so we see the actual command which will be run?

Reply:
changed to shell-quote call